README.adoc

```diff
@@ -0,0 +1,33 @@
+# Analysis Documents for WildFly
+
+[.lead]
+This repository is a central location for analysis and design documents of new features and capabilities in the https://wildfly.org[WildFly] project universe
+
+A WildFly Analysis Document (or "proposal") is a way to
+propose, communicate and coordinate on new efforts for the  project.
+
+This process is evolving, but is mandatory for all enhancements and requirements for WildFly.
+
+## Get Started
+
+1. Socialize an idea with others.  Make sure others think the work is worth doing, and are willing to review design and code changes required.
+2. Create a Pull Request with your new analysis document based on the link:./design-doc-template.adoc[template].
+
+## FAQs
+
+### Do I have to use the process?
+
+If this is an enhancement or new requirement for WildFly, yes.
```

Comment:
Perhaps

"If this introduces new user-controllable behavior to WildFly, yes.  Requirements not involving user controllable behavior that are of sufficient complexity should also use the process."

I want to avoid the word 'enhancement' because we use the JIRA Enhancement type for a different thing -- things that benefit the user but are not user controllable or otherwise require the feature process.

Reply:
Thanks for the feedback. I kept "yes" as the short story and further expand it based on your suggestion.